docs/ref/models/querysets.txt

```diff
@@ -1663,7 +1663,7 @@ For example::
 ``select_for_update()``
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-.. method:: select_for_update(nowait=False, skip_locked=False, of=())
+.. method:: select_for_update(nowait=False, skip_locked=False, of=(), no_key=False)
```

Comment:
Could you please also add a `.. versionchanged` annotation for `no_key`.

Reply:
Done